### ISSUE TITLE
Garbled screen on Centos 7 Virtual Machines (HyperV and Virtualbox)

### ISSUE THREAD

**Issue by marcinbojko** [dylanaraps/neofetch#1353]
## Description

Using default configuration terminal console output is unreadable. To restore it 'reset' command is needed (or restart)
If you're suggesting a new feature then just a description will suffice.

Problem can be resolved by changing color_blocks to 'off'
#### Neofetch version
6.1.0
## Screenshot
![Selection_560](https://user-images.githubusercontent.com/4726723/69655201-5a932700-1076-11ea-8751-12b8ede110bf.png)

## Config file
https://gist.github.com/marcinbojko/e4a668524158b1740f2227b7748d6897
## Verbose log
https://gist.github.com/marcinbojko/e4a668524158b1740f2227b7748d6897
1. Run `neofetch -vv 2> neofetchlog`
2. Upload the contents of `neofetchlog` to pastebin, gist or equivalent.

**Comment by dylanaraps** [comment]
Does this issue still occur in master?

**Comment by marcinbojko** [comment]
I don't know until you'll release some rpms

**Comment by dylanaraps** [comment]
Neofetch is a self contained file which requires no compilation. Just download and run it. You can test master by using the following link. It can also be run from anywhere. 

- https://raw.githubusercontent.com/dylanaraps/neofetch/master/neofetch

**Comment by marcinbojko** [comment]
Yes.
![image](https://user-images.githubusercontent.com/4726723/71559876-87ff3600-2a63-11ea-9c5f-da9ca2d21b17.png)
![image](https://user-images.githubusercontent.com/4726723/71559884-a8c78b80-2a63-11ea-91b4-699385406a42.png)

**Comment by dylanaraps** [comment]
Try the latest master once again. The same link as above will work. :+1:

**Comment by marcinbojko** [comment]
There is no change
![image](https://user-images.githubusercontent.com/4726723/71572272-8757b580-2ade-11ea-8e0e-58efee08eed6.png)

**Comment by dylanaraps** [comment]
Closing as stale. No one else can reproduce either.

**Comment by joshua-in-boots** [comment]
I have the same proble with my VM on VMware Fusion.

- OS: CentOS 7.9, installed with netinstall ISO.
- Packages installed by my choice
  - avahi
  - bash-completion
  - dnf
  - dnf-plugins-core

I installed neofetch as I descriibed below:
```
sudo yum -y install dnf dnf-plugins-core
sudo dnf update
sudo dnf copr enable konimex/neofetch
sudo dnf install neofetch
```

At first, I though the COPR repository may be the cause of this, but the same sequence with the neofetch by you, 
> https://raw.githubusercontent.com/dylanaraps/neofetch/master/neofetch

One thing funny is that I have no problem with my SSH sessions(pts/X), but I can see the odd one on my local console(tty1).
